packages/chakra-ui-core/src/CDivider/tests/CDivider.test.js

```diff
@@ -23,3 +23,12 @@ it('should change orientation', () => {
 
   expect(asFragment()).toMatchSnapshot()
 })
+
+it('should have corresponding aria-orientation attribute', () => {
+  const inlineAttrs = 'orientation="horizontal"'
+  const { asFragment } = renderComponent({ inlineAttrs })
+
+  const divider = screen.getByTestId('divider')
+  expect(divider).toHaveAttribute('aria-orientation', 'horizontal')
+  expect(asFragment()).toMatchSnapshot()
```

Comment:
we already have snapshot tests so maybe we can delete this one.

Reply:
Coolio.